openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -55,10 +59,28 @@
 )
 from . import _rfe_utils
 from ...utils import without_oechem_backend, log_system_probe
+from openfe.due import due, Doi
+
 
 logger = logging.getLogger(__name__)
 
 
+due.cite(Doi("10.5281/zenodo.1297683"),
```

Comment:
@mikemhenry I'm putting you in charge of adding any other citations for Perses that are relevant.

Reply:
tagging #600